src/AppBundle/Subscriber/AutoLabelPRFromContentSubscriber.php

```diff
@@ -25,7 +25,7 @@ public function __construct(CachedLabelsApi $labelsApi)
     public function onPullRequest(GitHubEvent $event)
     {
         $data = $event->getData();
-        if ('opened' !== $action = $data['action']) {
+        if (!in_array($action = $data['action'], array('opened', 'edited'), true)) {
```

Comment:
or a new value object for different types of events - e.g .`PullRequest` - with getters like `getAction()`, etc.